website/cue/reference/remap/functions/sha1.cue

```diff
@@ -15,7 +15,7 @@ remap: functions: sha1: {
 		},
 	]
 	internal_failure_reasons: []
-	return: types: ["string"]
+	return: types: ["integer"]
```

Comment:
I'm not clear why this was changed. I don't see the return type of the `sha1` function being changed?